quarkus/deployment/src/main/java/org/keycloak/quarkus/deployment/LiquibaseProcessor.java

```diff
@@ -35,11 +36,12 @@
 
     @Record(ExecutionTime.STATIC_INIT)
     @BuildStep
-    void configure(KeycloakRecorder recorder, CombinedIndexBuildItem indexBuildItem) {
+    void configure(KeycloakRecorder recorder, List<JdbcDataSourceBuildItem> jdbcDataSources, CombinedIndexBuildItem indexBuildItem) {
         DotName liquibaseServiceName = DotName.createSimple(LiquibaseService.class.getName());
         Map<String, List<String>> services = new HashMap<>();
-
         IndexView index = indexBuildItem.getIndex();
+        JdbcDataSourceBuildItem dataSourceBuildItem = jdbcDataSources.get(0);
```

Comment:
does it make sense to add a new local var here instead of just calling jdbcDataSources.get(0).getDbKind() one line below? Both is fine for me.

Reply:
It does, if you are OK leaving there I'll not change.

Reply:
sure, as said. :)